pandas/core/frame.py

```diff
@@ -2148,10 +2148,10 @@ def to_parquet(
             'pyarrow' is unavailable.
         compression : {'snappy', 'gzip', 'brotli', None}, default 'snappy'
             Name of the compression to use. Use ``None`` for no compression.
-        index : bool, default None
+        index : bool, default True
             If ``True``, include the dataframe's index(es) in the file output.
-            If ``False``, they will not be written to the file. If ``None``,
-            the behavior depends on the chosen engine.
+            If ``False``, they will not be written to the file.
+            If ``None``, RangeIndex will be stored as metadata-only.
```

Comment:
Maybe mention that other indexes are included as columns in the output in this case?